duo_client/admin.py

```diff
@@ -2876,3 +2876,96 @@ def sync_user(self, username, directory_key):
             '/admin/v1/users/directorysync/{directory_key}/syncuser').format(
                 directory_key=directory_key)
         return self.json_api_call('POST', path, params)
+
+    def get_trust_monitor_events_generator(
+        self,
+        mintime,
+        maxtime,
+        event_type=None,
+    ):
+        """
+        Returns a generator which yields trust monitor events.
+
+        Params:
+            mintime (int) - Return events that have a surfaced timestamp greater
+                            than or equal to mintime. Timestamp is represented as
+                            a unix timestamp in milliseconds.
+            maxtime (int) - Return events that have a surfaced timestamp less
+                            than or equal to maxtime. Timestamp is represented as
+                            a unix timestamp in milliseconds.
+            event_type (str, optional) - Limit the events returned by a supplied
+                                         event type represented as a string. If
+                                         not supplied, the caller will recieve all
+                                         event types. Check the Duo Admin API
+                                         documentation for expected values for
+                                         this parameter.
+
+        Returns: Generator which yields trust monitor events.
+        """
+
+        params = {
+            "mintime": "{}".format(mintime),
+            "maxtime": "{}".format(maxtime),
+        }
+
+        if event_type is not None:
+            params["type"] = event_type
+
+        return self.json_cursor_api_call(
+            "GET",
+            "/admin/v1/trust_monitor/events",
+            params,
+            lambda resp: resp["events"],
+        )
+
+    def get_trust_monitor_events(
```

Comment:
and while we're renaming could you change this one to `get_trust_monitor_events_by_offset` or something that clarifies the paging will need to be done manually by the user of this function?

Again I'm just requesting this for consistency since the other functions that are named something like
`get_duo_thing` do paging for you and return the entire list of data objects.

Reply:
Yup sure can! Should be updated now!